app/services/keyword_translator.rb

```diff
@@ -45,7 +45,7 @@ def translations
   end
 
   def translated_keyword
-    return keyword if translations.nil?
+    return keyword if translations.nil? || translations.length == 0
```

Comment:
Also, just for informational purposes, you can use `empty?` in Ruby to check if an array, string, or hash is empty.